server/datastore/mysql/app_configs_test.go

```diff
@@ -280,7 +280,6 @@ func testAppConfigDefaults(t *testing.T, ds *Datastore) {
 	require.Equal(t, 24*time.Hour, ac.WebhookSettings.Interval.Duration)
 	require.False(t, ac.WebhookSettings.HostStatusWebhook.Enable)
 	require.True(t, ac.HostSettings.EnableHostUsers)
-	require.True(t, ac.HostSettings.EnableScheduledQueryStats)
```

Comment:
Isn't that all `HostSettings` that have been removed? In the docs, the whole section was deleted.